Documentation/project-docs/uwp-compat.md

```diff
@@ -0,0 +1,489 @@
+# Behavior change between .NET Core and UWP
+
+This document contains list of not supported APIs on UWP but supported on .NET Core
+
+## Notes
+- This document is for preview only and was semi-generated, information may be incomplete or may contain errors
```

Comment:
I can move it over there when this review is finished although I find regular files easier to work with as you can i.e. send a PR for review